### PR TITLE
add add-item command

### PR DESCRIPTION
**Pull request by EthanVieira**
image
* Usage:
  /add-item name-of-item amount
  amount is optional and defaults to 1
  if amount is "max", set the amount to 2,147,483,647 if stackable or 28 if not stackable
  e.g.
 /add-item gold max
 /add-item BronzeDagger

### CODE REVIEW

File: src/constants/constants.js [hunk 1]
```diff
@@ -70,6 +70,8 @@ export const CONSTANTS = {
         MAX_LEVEL: 99,
         MAX_XP: 200_000_000,
         XP_FOR_99: 13_034_431,
+        MAX_ITEM_STACK: 2_147_483_647,
```
Comment (CurtisGreen) [inline src/constants/constants.js:73]: lol

File: src/utilities.js [hunk 2]
```diff
@@ -33,9 +33,12 @@ export function calcXpForLevel(lv) {
...
 var itemClasses = {};
 export async function getItemClass(itemName, scene) {
-    let itemClass = itemClasses[itemName];
-
-    return new itemClass.default(scene);
+    if (itemClasses.hasOwnProperty(itemName)) {
+        let itemClass = itemClasses[itemName];
+        return new itemClass.default(scene);
+    } else {
+        return null;
```
Comment (CurtisGreen) [inline src/utilities.js:41]: I think we should log if we're passing in a bad item name here

**Review by CurtisGreen — COMMENTED** [review]
I thought the INT max would be a problem but it handles it like a champ lol

File: src/utilities.js (hunk 2 above)
Comment (EthanVieira) [reply src/utilities.js:41]: Updated! https://github.com/EthanVieira/rs-clicker/pull/79/commits/7291bf8df46a141986148de968f0b8284f32358a

File: src/utilities.js [hunk 3]
```diff
@@ -33,9 +33,13 @@ export function calcXpForLevel(lv) {
...
 var itemClasses = {};
 export async function getItemClass(itemName, scene) {
-    let itemClass = itemClasses[itemName];
-
-    return new itemClass.default(scene);
+    if (itemClasses.hasOwnProperty(itemName)) {
+        let itemClass = itemClasses[itemName];
+        return new itemClass.default(scene);
+    } else {
+        console.log("Invalid item name in getItemClass - returning null.");
```
Comment (CurtisGreen) [inline src/utilities.js]:
```suggestion
        console.log("Invalid item name in getItemClass - returning null.", itemName);
```

**Review by CurtisGreen — APPROVED** [review]

**Pull request merged**